### DISCUSSION TITLE
Enhanced display of linked Pull Requests

### DISCUSSION THREAD

**Discussion by zbjornson** [community/community#10252] [Projects and Issues]
Since [Dec 17, 2021](https://github.com/github/feedback/discussions/5003#discussioncomment-1836844), linked PRs show up as "pills" in issue cards. There are a few limitations with this:

* There's no way to see info about the PR, such as status/checks, labels, assignees, milestone, review status.
* If both PRs and issues are added to a project, both cards show up and clutter up the board. If the PRs were instead nested under the issue, that would avoid the clutter. The issue linked above has a workaround of only adding issues and not PRs to the project, but that would require custom automation that still adds PRs that don't have corresponding issues.

I'm proposing to replace the "pill" with nesting as shown in the screenshots in https://github.com/github/feedback/discussions/5003#discussioncomment-1655004:

![image](https://user-images.githubusercontent.com/469365/149986545-ef10ff36-bd9a-483f-b86c-519cb6f786c1.png)
![image](https://user-images.githubusercontent.com/469365/149986561-a9515bff-a7b2-4867-b92c-f9d3e4fd930a.png)
![image](https://user-images.githubusercontent.com/469365/149986610-ff4f6832-8fd3-4adc-b540-e9c26d5a0d5a.png)

**Comment by benbruscella** [comment]
Adding to this...

Issues currently have the option of showing the "Linked Pull Request".

Could we please get the PR's showing the "Linked Issues"? 

That would mean we could create 2 views, one with `is:issue` the other with `is:pr` and have links to both the issue and the PR on both views.
